src/features/defi/providers/thorchain-savers/components/ThorchainSaversManager/Withdraw/components/Withdraw.tsx

```diff
@@ -180,9 +183,20 @@ export const Withdraw: React.FC<WithdrawProps> = ({ accountId, fromAddress, onNe
       asset,
       accountId,
       amountCryptoBaseUnit: toBaseUnit(cryptoAmount, asset.precision),
-      enabled: hasEnoughStakingBalance,
+      enabled: hasEnoughStakingBalance && !isRunePool,
     })
 
+  const memo = useMemo(() => {
+    const balanceCryptoPrecision = bnOrZero(amountAvailableCryptoPrecision.toPrecision())
+    const percent = bnOrZero(cryptoAmount).div(balanceCryptoPrecision).times(100).toFixed(0)
+    const basisPoints = convertPercentageToBasisPoints(percent)
+
+    if (isRunePool) return `pool-:${basisPoints}`
+    if (thorchainSaversWithdrawQuote?.memo) return thorchainSaversWithdrawQuote.memo
```

Comment:
Same thing here re: returning this first and keep the meaty logic after

Reply:
Done in [1f9d825](https://github.com/shapeshift/web/pull/7449/commits/1f9d8254294f6843bde971347039e344909ac032)